ui-frontend/packages/catalog-ui-search/src/main/webapp/component/pages/incompatible-browser.less

```diff
@@ -0,0 +1,53 @@
+@import '../../styles/vars.less';
```

Comment:
I put this import here so I could use predefined settings even though I'm not running `ApplicationSetup.js`.

Reply:
I think we should inline the styles if possible (feel free to hardcode the values rather than use the theming) since that's most likely to be compatible.

Reply:
I added inlined styles for IE 9 and below, but IE 10 and 11 are still able to use this.